app/backend/registries/urls.py

```diff
@@ -73,7 +73,9 @@
         views.PersonNoteDetailView.as_view(), name='person-note-detail'),
 
     # Person endpoints (drillers, well installers, and other instances of Person model)
-    url(r'^api/v1/drillers/names/$',
+    # TODO: There's some confusion between drillers and persons. Sometimes we're looking only for drillers,
+    # sometimes we're actually looking for people (pump installers, drillers etc.)
+    url(r'^api/v1/persons/names/$',
```

Comment:
I agree with the comment here but this has been discussed and for whatever reason we've settled on `drillers`.  We also published the endpoint`api/v1/drillers/names` so we can't change it.